app/src/main/java/com/hover/stax/home/MainActivity.kt

```diff
@@ -344,6 +345,12 @@ class MainActivity : AbstractNavigationActivity(), BalancesViewModel.RunBalanceL
                     .extra(HoverAction.NOTE_KEY, transferViewModel.note.value)
                     .extra(Constants.ACCOUNT_NAME, selectedAccount?.name)
 
+            if(nonTemplateVariables!=null && nonTemplateVariables.isNotEmpty()) {
```

Comment:
You can use `!isNullOrEmpty()` to simplify the condition.

Reply:
Done, thanks.